### PR TITLE
Use Firefox-specific CSS to draw background-image gradient on progress bar

### PR DESCRIPTION
**Pull request by JustinPinner**
Restores background gradient fill on progress bar for Firefox browsers.

**Before:**
Screen Shot 2019-03-20 at 14 11 16 - FF - BEFORE

**After:**
Screen Shot 2019-03-20 at 14 11 06 - FF - AFTER

Also re-tested (locally) with Chrome and Safari and they're still ok.

### CODE REVIEW

File: core/src/main/resources/css/atoms/_audio.scss [hunk 1]
```diff
@@ -181,15 +181,15 @@ input[type=range]:focus {
 
 .atom--audio__progress-bar input[type=range]::-moz-range-progress {
   .garnett--pillar-news      &,
-  .content--pillar-news      & { background-color: linear-gradient(to right, $c-pillar--news 0%, #afafaf 0%); }
+  .content--pillar-news      & { background-image: linear-gradient(to right, $c-pillar--news 100%, #afafaf); }
```
Comment (regiskuckaertz) [inline core/src/main/resources/css/atoms/_audio.scss]: How could I miss that 😵 Sorry.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/atom-renderer",
-  "version": "0.17.5",
+  "version": "0.17.6",
```
Comment (regiskuckaertz) [inline package.json]: This will be 0.18.1 instead.

Comment (JustinPinner) [reply package.json]: Yep, that should have changed after the merge master -> resolve conflicts step

Comment (JustinPinner) [reply package.json]: And it has ✔️

File: core/src/main/resources/css/atoms/_audio.scss (hunk 1 above)
Comment (JustinPinner) [reply core/src/main/resources/css/atoms/_audio.scss]: No worries, I _wrote_ it and didn't see it! 🙄

**Pull request merged**